### PR TITLE
added test-cases for adding coment

### PR DESCRIPTION
**Pull request by virtus80**
- modified description of positive test-cases for article title and article with valid content
- added positive and negative test-cases for adding comment to existing article

### CODE REVIEW

File: functional-tests-antarcticle/src/test/groovy/org/jtalks/tests/antarcticle/ArticleManagementTest.groovy [hunk 1]
```diff
@@ -105,5 +105,29 @@ class ArticleManagementTest extends Specification {
       'second'        | $randomLanguage     | 'Could not create second translation of article'
       'same language' | 'English'           | 'Could not create translation of article on same language'
   }
+  
+  def 'comment adding positive cases'() {
+    given: 'user is created and logged in'
+        and: 'article is created and opened'
```
Comment (ctapobep) [inline functional-tests-antarcticle/src/test/groovy/org/jtalks/tests/antarcticle/ArticleManagementTest.groovy]: Formatting is off

File: functional-tests-antarcticle/src/test/groovy/org/jtalks/tests/antarcticle/ArticleManagementTest.groovy [hunk 2]
```diff
@@ -105,5 +105,29 @@ class ArticleManagementTest extends Specification {
...
+      'Min boundary'  | randomAlphabetic(1)     | 'Could not create comment with min length boundary'
+      'Max boundary'  | randomAlphabetic(65535) | 'Could not create comment with max length boundary'
+      'Average length'| randomAlphabetic(3000)  | 'Could not create comment of average length'
+  }
+  
+def 'comment adding negative cases'() {
+    given: 'user is created and logged in'
+        and: 'article is created and opened'
+    when: "user creates a comment with $caseName"
+    then: 'comment should not get added'
```
Comment (ctapobep) [inline functional-tests-antarcticle/src/test/groovy/org/jtalks/tests/antarcticle/ArticleManagementTest.groovy]: then: 'validation error should be shown'

**Comment by ctapobep** [comment]
Could you please squash the commits into one? Google for "git squash with reset soft"

**Comment by ctapobep** [comment]
@virtus80 why did you create another PR while this one is not merged yet?

**Comment by virtus80** [comment]
@ctapobep I used 'revert'. I thought that it cancelled last commit.

**Comment by ctapobep** [comment]
Why did you decide to revert?

File: functional-tests-antarcticle/src/test/groovy/org/jtalks/tests/antarcticle/ArticleManagementTest.groovy [hunk 3]
```diff
@@ -105,5 +105,29 @@ class ArticleManagementTest extends Specification {
...
+    when: "user creates a comment with $caseName"
+    then: 'comment should be added'
+    where:
+      caseName        | commentLength           | messageIfCaseFailed
+      'Min boundary'  | randomAlphabetic(1)     | 'Could not create comment with min length boundary'
+      'Max boundary'  | randomAlphabetic(65535) | 'Could not create comment with max length boundary'
+      'Average length'| randomAlphabetic(3000)  | 'Could not create comment of average length'
+  }
+  
+def 'comment adding negative cases'() {
```
Comment (ctapobep) [inline functional-tests-antarcticle/src/test/groovy/org/jtalks/tests/antarcticle/ArticleManagementTest.groovy]: Formatting is off!

**Pull request closed without merging**